### DISCUSSION TITLE
Consulta Enunciado TP-1 Ej. 2 Conversion de Temperaturas

### DISCUSSION THREAD

**Discussion by martinjguevara** [INGCOM-UNRN-P1/cursada-2023#40] [Preguntas y respuestas] [answered]
Por consola tengo que ingresar 2 valores? uno en Celsius que se conviertan a Fahrenheit y otro en Fahrenheit que se convierta a Celsius?
O solo ingreso un valor en Celsius que se convierte a Fahrenheit? Porque el enunciado indica dos funciones pero no le encuentro sentido a ingresar un valor en Celsius y que se devuelva en Fahrenheit y nuevamente en Celsius...

**Comment by martinvilu** [comment, marked as answer]
Ambos caminos son correctos.

Con un valor podes "ir y volver" o podes pedir dos
